app/models/user.rb

```diff
@@ -12,15 +12,14 @@ class User < ActiveRecord::Base
   self.table_name = 'rusers'
   alias_attribute :name, :username
 
-  NORMAL = 1 # Usage: User::NORMAL
-  BANNED = 0 # Usage: User::BANNED
-  MODERATED = 5 # Usage: User::MODERATED
```

Comment:
Here i'd like to preserve these to show people how to use these states, if that's OK? Thanks!!